sycl/test/regression/macro_conflict.cpp

```diff
@@ -0,0 +1,25 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+// expected-no-diagnostics
+//
+//==-------------- macro_conflict.cpp --------------------------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
```

Comment:
AFAIK, no need to include copyright headers into tests

Reply:
Okay, I will remove it. I just copied from some other regression test case.